pkg/cloudprovider/providers/vsphere/vclib/volumeoptions.go

```diff
@@ -0,0 +1,80 @@
+package vclib
+
+import (
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+// VolumeOptions specifies various options for a volume.
+type VolumeOptions struct {
+	CapacityKB             int
+	Tags                   map[string]string
+	Name                   string
+	DiskFormat             string
+	Datastore              string
+	VSANStorageProfileData string
+	StoragePolicyName      string
+	StoragePolicyID        string
+	SCSIControllerType     string
+}
+
+var (
+	diskFormatValidType = map[string]string{
```

Comment:
Can you make diskFormatValidType public?

We are planning to add diskmanager sub pakcage, there we should be able access this using 
vclib.DiskFormatValidType